Python/Tests/Core/DebugReplEvaluatorTests.cs

```diff
@@ -63,6 +65,7 @@ public class DebugReplEvaluatorTests {
             _window = new MockReplWindow(_evaluator);
             _evaluator._Initialize(_window);
             _processes = new List<PythonProcess>();
+            _moduleFileExtension = Version.Version < Microsoft.PythonTools.Parsing.PythonLanguageVersion.V30 && Version.IsCPython ? ".pyc" : ".py";
```

Comment:
As this is class-specific state, it would be better as a property than an instance variable.

At the very least, the instance variable should be readonly. Arguably we should also fix the module file extension to be corrected to `.py` wherever possible as well, which would presumably also make this redundant.